internal/apiserver/route_get_identity_by_id.go

```diff
@@ -20,17 +20,19 @@ import (
 	"net/http"
 	"strings"
 
+	"github.com/google/uuid"
```

Comment:
We don't use the google UUID library directly, use the one in firefly-common

Reply:
Same here, we are not validating the input, rather we are just filtering the input, so I didn't think we need to call the UUID validator with the context.

Reply:
just more of a consistency and dependency thing to use the firefly-common library for this